src/elements/Reviews.jsx

```diff
@@ -0,0 +1,24 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getReviews } from "../utility/axios-request";
+import ReviewCard from "./reviews_li_card";
+
+function Reviews() {
+  const [reviewState, setReviewState] = useState([]);
+  useEffect(() => {
+    getReviews().then((response) => {
+      setReviewState(response);
+    });
```

Comment:
It would be good to set up a loading screen here as well as in your single review page, so that users can see when your app is fetching the list of reviews

Reply:
will do